src/Auth/Auth.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import { RouteComponentProps } from "react-router-dom";
+
+import Button from "src/components/Button";
+import Login from "src/Auth/Login";
+import SignUp from "src/Auth/SignUp";
+import {
+  AuthContainer,
+  AuthenticationWrapper,
+  OverlayContainer,
+  Overlay,
+  OverlayLeft,
+  OverlayRight,
```

Comment:
I think these belong inside `src/components`

Reply:
No these components are Auth specific. If we need them for other places we can later modify them accordingly and put them in `src/components` right now they are pretty auth specific.

Reply:
We'll add a `"Auth"` prefix to these